src/main/java/core/mate/academy/service/producers/TruckProducer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.service.producers;
+
+import core.mate.academy.model.Truck;
+import core.mate.academy.service.MachineProducer;
+import java.util.Arrays;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    @Override
+    public List<Truck> get() {
+        return Arrays.asList(new Truck("Truck1", "Red", 5),
+                new Truck("Truck2", "Red", 15));
```

Comment:
Follow one codestyle 
```suggestion
        return Arrays.asList(
                new Truck("Truck1", "Red", 5),
                new Truck("Truck2", "Red", 15));
```